app/src/parser_print.c

```diff
@@ -440,6 +440,11 @@ static const key_subst_t key_substitutions[] = {
     {"msgs/value/funds", "Funds"},
     {"msgs/value/msg", "Msg"},
     {"msgs/value/sender", "Sender address"},
+
+    {"msgs/value/inputs/address", "Source Address"},
```

Comment:
Awesome 😎 I'm not sure if we should use `Src Address` or something shorter for titles but this looks really nice 💪